Dockerfile

```diff
@@ -22,6 +22,13 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+FROM python:slim
+RUN pip install build
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends git
+COPY . /src/fmriprep
+RUN python -m build /src/fmriprep
```

Comment:
I've tested locally. Modifying the repository will make this rerun but will not cause a cache miss on L33+ until there's another change in the Dockerfile or a `COPY`/`ADD` command fails a hash check.